org-roam.el

```diff
@@ -292,6 +355,15 @@ Titles are obtained via:
         (cons title alias-list)
       alias-list)))
 
+(defun org-roam--extract-and-format-titles (&optional file-path)
+  "Extract the titles from the current buffer and format them.
+If FILE-PATH is not provided, the file associated with the
+current buffer is used."
+  (let ((titles (org-roam--extract-titles)))
```

Comment:
I think we can use dash to clean this up.

```
(-> (org-roam--extract-titles) (org-roam--format-title file-path))
```

Apologies for the poor formatting, typing elisp on github.